### PR TITLE
Check Answer Fix

### PR DESCRIPTION
**Pull request by satchi5**
- Fixed user answers returning as incorrect when user has included an article in their answer in RiddleRush, Trivia and MysteryWord
- Fixed user answers matching Skogs returning as an invalid answer in MindReader
+ Added questions to WouldYouRather and MindReader

### CODE REVIEW

File: src/events/ready/daily-games.js [hunk 1]
```diff
@@ -12,4 +12,4 @@ module.exports = async (c) => {
         const selectedGame = games[Math.floor(Math.random() * games.length)];
         selectedGame(c);
     });
-};
\ No newline at end of file
+    };
```
Comment (Lewwy95) [inline src/events/ready/daily-games.js]: The indentation for this is a little off, you could disregard the file if that's easier!

**Review by Lewwy95 — CHANGES_REQUESTED** [review]
Everything looks ok! If you could disregard the daily-games.js file I will merge it together - thank you.

**Review by satchi5 — COMMENTED** [review]
Fixed indentation in line 15 of daily-games.js :)

**Review by Lewwy95 — APPROVED** [review]

File: src/events/ready/daily-games.js (hunk 1 above)
Comment (satchi5) [reply src/events/ready/daily-games.js]: See comment below

**Pull request merged**